zerozeroreadme/pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = "^3.11"
 ruamel-yaml = "^0.18.5"
 toml = "^0.10.2"
 tomli_w = "^1.0"
+preflight_parser = {git = "https://github.com/arXiv/submission-tools.git", subdirectory = "preflight_parser"}
```

Comment:
Don't you need a version here?

Reply:
Not here, this is for general development and should pick the latest version. 
In gen2pdf, which is baked into containers, we will have a versioned dependency (probably).